sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -76,50 +75,69 @@ export interface ChatClientOptions extends PipelineOptions {
 }
 
 // @public
-export interface ChatMessage extends Omit<RestChatMessage, "senderId" | "content"> {
+export interface ChatMessage {
     content?: ChatMessageContent;
-    sender?: CommunicationUserIdentifier;
+    createdOn: Date;
+    deletedOn?: Date;
+    editedOn?: Date;
+    id: string;
+    senderCommunicationIdentifier?: CommunicationIdentifierModel;
+    senderDisplayName?: string;
+    sequenceId: string;
+    type: ChatMessageType;
+    version: string;
 }
 
-// @public (undocumented)
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
+// @public
+export interface ChatMessageContent {
+    initiatorCommunicationIdentifier?: CommunicationIdentifierModel;
+    message?: string;
     participants?: ChatParticipant[];
```

Comment:
Why does the content have an array of participants on it?

Reply:
@juancamilor  @amrelroumy-msft

Reply:
similar to `initiator` this would be present on system messages, for example `participantAdded` message would contain the participants added